src/colors.tsx

```diff
@@ -15,6 +15,7 @@
  */
 export const background = '#F6F7F9';
 export const primary = '#681DA8';
+export const secondary = '#1B3A57';
```

Comment:
Can you add a comment with the Material color?

Reply:
i.e. `// MATERIAL_GREY_200`

Reply:
Add a few comments. It actually took me a while to realize that this is a Firebase color, not a Material one.